public/scripts/controllers/BooksShowController.js

```diff
@@ -0,0 +1,49 @@
+angular
+  .module("angularBooks")
+  .controller("BooksShowController", BooksShowController);
+
+BooksShowController.$inject = ["$http", "$routeParams", "$location"];
+
+function BooksShowController(   $http,   $routeParams, $location ) {
```

Comment:
real subtle, ocd type comment here, but it looks cleaner/is easier to read when the '$location's line up